chef/cookbooks/crowbar-pacemaker/recipes/drbd.rb

```diff
@@ -25,15 +25,24 @@
 claimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.claimed(node, claim_string).sort
 
 if claimed_disks.empty? and not unclaimed_disks.empty?
+
+  disk_name_to_claim = node["crowbar_wall"].disks_roles.find { |_, role| role == claim_string }[0] rescue ""
```

Comment:
Hm, true.
I'm not sure though. I thought about this more like a pre-claiming, which could be still changed (unlike the disks that are already claimed). Than I'm moving the definite point to later, the actual barclamp that's gonna need it.